src/stores/walletStore.ts

```diff
@@ -82,6 +87,17 @@ export class WalletStore {
     return newWallet;
   };
 
+  removeApiWallet = async (wallet: WalletModel) => {
+    const {id} = wallet;
+    try {
+      return await this.api!.removeApiWallet(id);
+    } catch (reason) {
+      return Promise.reject(reason);
+    } finally {
+      this.removeWallet(wallet);
```

Comment:
but you _don't need to remove_ the wallet from the list in a case of failure, plz add a test for that

Reply:
yes, also I will show error in uiStore